src/components/LoginPage/LoginPageComponent.js

```diff
@@ -89,7 +89,9 @@ const Button = css`
 
 const GoogleLoginButton = styled.button`
   ${Button}
-  float: right;
+  position: absolute;
+  right: 2rem;
+  top: 1rem;
```

Comment:
What is this for? Did we have any problems with the login button on the login page?

@TomaTomoe pls review.

Reply:
It adds a little margin on top